packages/frontend/.eslintrc.json

```diff
@@ -1,13 +1,6 @@
 {
   "root": true,
```

Comment:
We can remove this and then remove `"../../.eslintrc.json"` from `"extends"`, right? The root `eslintrc` will then be merged automatically.

Reply:
I believe so:
https://eslint.org/docs/latest/user-guide/configuring/configuration-files#cascading-and-hierarchy

I also thinks that's usually the way to do it.